docs/users/hermez-wallet.md

```diff
@@ -60,15 +60,18 @@ Opening a transaction shows information related to that transaction. There's als
 
 ## Withdrawals
 
-Withdrawals are a two-part process. After completing the first part explained above, a card appears on the Home screen or on the respective account page. When ready, it will show a button to finalize the withdrawal.
+Withdrawals are a two-part process. The first part requires to select the Token you want to withdraw from (for example HEZ) and in the next screen click on `Withdraw` button, enter the amount to withdraw and click on `Continue`.
 
+After completing the first part explained above, a card appears on the Home screen or on the respective account page. When ready, it will show a button to finalize the withdrawal.
```

Comment:
```suggestion
After completing the first part explained above, a card with the withdrawal details appears on the Home screen or on the respective account page. When ready, it will show a button to finalize the withdrawal.
```